pandas/tests/series/test_dtypes.py

```diff
@@ -94,6 +94,14 @@ def cmp(a, b):
         result = ser.astype("object").astype(CategoricalDtype())
         tm.assert_series_equal(result, roundtrip_expected)
 
+    def test_categorical_int_to_int32(self):
```

Comment:
should be named test_categorical_astype_to_int

Reply:
Done